tests/unit/test_util.py

```diff
@@ -197,4 +193,3 @@ def test_url_gets_to_stdout(self):
     if "-v" in sys.argv:
         debug = True
     pybamm.settings.debug_mode = True
```

Comment:
Ideally, we should set this `pybamm.settings.debug_mode = True` by default when running the tests, perhaps in `conftest.py` as a fixture (with `autouse` set to `True`) in your other PR.